pkg/controller/restore_controller_test.go

```diff
@@ -67,13 +68,15 @@ func TestFetchBackupInfo(t *testing.T) {
 		{
 			name:              "lister has backup",
 			backupName:        "backup-1",
+			namespace:         "velero",
 			informerLocations: []*velerov1api.BackupStorageLocation{builder.ForBackupStorageLocation("velero", "default").Provider("myCloud").Bucket("bucket").Result()},
 			informerBackups:   []*velerov1api.Backup{defaultBackup().StorageLocation("default").Result()},
 			expectedRes:       defaultBackup().StorageLocation("default").Result(),
 		},
 		{
 			name:              "lister does not have a backup, but backupSvc does",
 			backupName:        "backup-1",
+			namespace:         "velero",
```

Comment:
Not sure if really required, but maybe we can test a namespace other than the default "velero".

Reply:
Yes that would be possible, actually this test was a first naive attempt to also make all tests pass again. This claim has been discarded since then :wink:, because many test now fail because of the changes. At the moment, there is aspiration to fix all the tests. Should be done via the follow-up cleanup PR.